addons/binding/org.openhab.binding.nibeheatpump/README.md

```diff
@@ -0,0 +1,868 @@
+# Nibe heat pump binding
+
+The Nibe heat pump binding is used to get live data from from Nibe heat pumps without using expensive MODBUS40 adapter. This binding should be compatible with at least the F1145 and F1245 heat pump models.
```

Comment:
Could you please enter a newline after every sentence? (Won't make any difference to the rendering, but it simplifies future diffs for git).

Reply:
Done?